src/components/ui/button/button.scss

```diff
@@ -5,29 +5,29 @@
 
 /** Variables **/
 
-$color-success: #31e27b;
+$color-success: #{$green-300};
 $color-secondary-active: rgba(0, 0, 0, 0.12);
 $timing-primary: 0.2s;
 $secondary-underline-height: 2px;
 $secondary-underline-offset: 8px;
 $secondary-underline-offset-hover: 4px;
 
 .kui-theme--light {
-  --color-button--active: #{rgba($color-dark, 0.85)};
-  --color-button--border: #{$color-dark};
-  --color-button--border--disabled: #{rgba($color-dark, 0.3)};
-  --color-button__text--hover: #{rgba($color-light, 0.85)};
-  --color-button__text: #{rgba($color-dark, 0.85)};
-  --color-button__text--disabled: #{rgba($color-dark, 0.3)};
+  --color-button--active: #{$black-800};
+  --color-button--border: #{$black-900};
+  --color-button--border--disabled: #{rgba($color-dark, 0.3)}; // (#000000, 0.3) => $black-600 on Figma - Typography
```

Comment:
hey @cvetankanechevska I'm not 100% clear with the comment here. Does it mean the `#{rgba($color-dark, 0.3)}` should be replaced with `$black-600`?

Reply:
All rgba values are replaced with the new HEX.